tests/behat/features/create-taxonomies.feature

```diff
@@ -38,6 +38,6 @@ Feature: Create taxonomies
     And I press the "Create" button
     When I follow "My taxonomy term"
     And I click the "Children" CMS tab
-    Then I should see "My child taxonomy term"
+    Then I should see "My child taxonomy term" in the "#Form_ItemEditForm_Children" element
     When I follow "Taxonomy Terms"
-    Then I should not see "My child taxonomy term"
+    Then I should not see "My child taxonomy term" in the "#Form_EditForm_SilverStripe-Taxonomy-TaxonomyTerm" element
```

Comment:
```suggestion
    Then I should not see "My child taxonomy term" in the "##Form_ItemEditForm_Children" element
```
We still need to use the same element in both places

Reply:
I'm not sure that it's possible. This is two different forms Tab "Children" in "Taxonomy Terms" and "Taxonomy Terms" itself. Probably, I don't understand what do you want to achieve.

Reply:
I repeated all steps manually. It's not possible. Could you, please, explain what is your expectation? Probably, I do something wrong. Thank you.

Reply:
Oh, I didn't know they're two different tabs. Might be a good idea to add a quick inline comment above the second one explaining that for people like me who just glance at it.